packages/server/api/src/app/flows/step-run/step-run-service.ts

```diff
@@ -48,7 +48,7 @@ export const stepRunService = {
       projectId,
     });
 
-    if (step.id) {
+    if (step.id && result.output) {
       await flowStepTestOutputService.save({
```

Comment:
will it be a problem that we don't save it for this step? what is going to happen when you run a workflow that has it, and try to view it in runs pane?

Reply:
ok so apparently the run workflow panel takes the output from flow-run, not test-output, now for the output of test step it will just be a {} if it cannot find it in the DB